backend/test/akvo/flow_services/end_to_end_test.clj

```diff
@@ -203,6 +203,7 @@
     (mock-flow-report-api flow-report-id)
     (mock-gae survey-id)
     (test-util/mock-mailjet)
+    (test-util/mock-sentry)
```

Comment:
I now remember that I did not setup the mock sentry because if we do, if the test fails due to an exception, this exception is **just** going to be sent to the Sentry mock, so basically it is going to disappear in the WireMock ether, leaving the Dev clueless about what happened.

Maybe add an assert in this particular test to check that there were no exceptions sent to Sentry (and log the actual exception sent to Sentry to help the dev figure out the issue) or do not mock Sentry for this particular test.